@xen-orchestra/proxy/src/app/mixins/remotes.js

```diff
@@ -34,7 +35,10 @@ export default class Remotes {
         ],
 
         test: [
-          ({ remote }) => using(this.getHandler(remote), handler => handler.test()),
+          ({ remote }) =>
+            using(this.getHandler(remote), handler => handler.test()).catch(error => {
+              throw new JsonRpcError(error.message, error.code)
```

Comment:
What is `error.code` here?

Reply:
If the `error` has a code, we passe it to the client. It will simplify the error test in the client side.